lib/application.js

```diff
@@ -127,28 +131,8 @@ module.exports = class Application extends Emitter {
     };
   }
 
-  /**
-   * Initialize a new context.
-   *
-   * @api private
-   */
-
   createContext(req, res) {
```

Comment:
true. again, did it for reverse compatibility, but it doesn't make sense to keep it, you're right